AWSPinpointTests/AWSPinpointTargetingClientTests.m

```diff
@@ -79,19 +80,21 @@ - (AWSPinpointConfiguration *)getDefaultAWSPinpointConfiguration {
 - (AWSPinpointConfiguration *)getAWSPinpointConfigurationWithOptOut:(BOOL)optOut {
     AWSPinpointConfiguration *configuration = [self getDefaultAWSPinpointConfiguration];
 
-    [self setApplicationLevelOptOut:configuration withOptOut:optOut];
-
-    return configuration;
-}
-
-- (void)setApplicationLevelOptOut:(AWSPinpointConfiguration *)configuration withOptOut:(BOOL)optOut {
     configuration.isApplicationLevelOptOut = ^BOOL{
         return optOut;
     };
+
+    return configuration;
 }
 
-- (void)setApplicationLevelOptOut:(BOOL)optOut {
-    [self initializePinpointWithConfiguration:[self getAWSPinpointConfigurationWithOptOut:optOut]];
+- (void)setDeviceTokenInUserDefaults {
+    const unsigned char currentTokenBytes[] = {
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+        0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f
+    };
+    NSData *currentTokenData = [[NSData alloc] initWithBytes:currentTokenBytes length:16];
```

Comment:
If i think back to my C-programming days, i think this should be:
```
    NSData *currentTokenData = [[NSData alloc] initWithBytes:currentTokenBytes length:sizeof(currentTokenBytes)/sizeof(unsigned char)]
```

Reply:
oh, this seems to be copy and pasted across the entire class.. hmm.. leaving this as 16 is probably fine.